app/views/hq/orphans/_form.html.erb

```diff
@@ -0,0 +1,224 @@
+<%= form_for [:hq, orphan] do |f| %>
+  <%= render 'shared/errors', object: f.object %>
+  <div class="panel panel-default">
+    <div class="panel-heading">
+      <h3 class="panel-title">Orphan Details</h3>
+    </div>
+    <div class="panel-body">
+      <div class="form-group">
+        <%= f.label :name %>
+        <%= f.text_field :name, class: "form-control" %>
+      </div>
+      <div class="form-group">
+        <%= f.label :date_of_birth %>
+        <%= f.text_field :date_of_birth, class: "form-control date-picker", value: orphan.date_of_birth %>
+      </div>
+      <div class="form-group">
+        <%= f.label :gender %>
+        <%= f.select :gender,
+          Settings.lookup.gender, {}, class: "form-control" %>
+      </div>
+      <div class="form-group">
+        <%= f.label :health_status %>
+        <%= f.text_field :health_status, class: "form-control" %>
+      </div>
+      <div class="form-group">
+        <%= f.label :schooling_status %>
+        <%= f.text_field :schooling_status, class: "form-control" %>
+      </div>
+      <div class="form-group">
+        <%= f.label :goes_to_school %>
+        <%= f.check_box :goes_to_school %> 
+      </div>
+      <div class="form-group">
+        <%= f.label :orphan_status_id %>
+        <%= f.collection_select :orphan_status_id,
+          orphan_statuses, :id, :name, {}, 
+          {class: "form-control"} %>
+      </div>
+      <div class="form-group">
+        <%= f.label :orphan_sponsorship_status_id %>
+        <%= f.collection_select :orphan_sponsorship_status_id,
+          orphan_sponsorship_statuses, :id, :name, {}, 
+          {class: "form-control", disabled: "true"} %>
+      </div>
+      <div class="form-group">
+        <%= f.label :priority %>
+        <%= f.select :priority, 
+          %w(Normal High), {}, class: "form-control" %>
+      </div>
+      <div class="form-group">
+        <%= f.label :record_imported_on %>
```

Comment:
Here and on L55, this isn't quite how you want to handle the label change. When the attribute for `f.label` and the attribute for the corresponding `f.text_field` match, a user can click on either of them to make the input active and to start entering data. Passing a different attribute to the label than to the input breaks this behaviour.

What you want to do instead is keep the attribute but pass custom text for the label like so:
```erb
<%= f.label, :created_at, 'Record imported on' %>
```

The input is disabled here and so this point is entirely academic, but it's preferable to stick to conventions and best practices nonetheless.